src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -83,10 +88,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * Loads the Event Dispatcher from the container.
      *
+     * @param string $serviceId
+     *
      * @return EventDispatcherInterface
      */
-    protected function getEventDispatcher()
+    protected function getEventDispatcher($serviceId)
     {
-        return $this->getContainer()->get('event_dispatcher');
+        $eventDispatcher = $this->getContainer()->get($serviceId);
+
+        if (false === $eventDispatcher instanceOf EventDispatcherInterface) {
+            throw new \RuntimeException(sprintf('The service id "%s" does not reference to an event dispatcher', $serviceId));
```

Comment:
I'd say "[...] does not reference an event dispatcher." (remove "to" and add a dot at the end).